frontend/.env.development

```diff
@@ -15,3 +15,4 @@ SENDY_API_URL=
 SENDY_LIST_ID=
 
 NEXT_PUBLIC_API_URL=http://localhost:8080
+NEXT_PUBLIC_LOCAL_AUTH_TOKEN=LOCAL_AUTH_12345678
```

Comment:
hardcode public auth key for local to local calls

Reply:
Add this as a comment into the file itself, it'll be easier for auditors to find it this line gets flagged

Reply:
Thanks! added.

And for add'l context - `.env.development` just works for local dev (`npm run dev`)
We will need to update how the API is called so that we can pull in actual auth keys from AWS parameter store in lower and prod envs